scripts/20220221_smugmug-activity.sql

```diff
@@ -19,6 +18,5 @@ CREATE TABLE IF NOT EXISTS `smugmug_uploads` (
     gallery_slug VARCHAR(150),
     photo_slug VARCHAR(150),
     created_at datetime,
-    INDEX (`created_at` DESC),
-    UNIQUE INDEX `content_unique` (`gallery_slug`, `photo_slug`)
+    INDEX (`created_at` DESC)
```

Comment:
The results sorting becomes non-deterministic when two photos have the exact same upload time. This should only be an issue with batch uploads, but it does mean that we might not properly figure out where to stop looking through the feed (if the last photo in the db is mid-batch) and try to insert things that are already there. To avoid unique constraint errors, I'm just going to remove it because I think we can do without absolute perfection here :)

Same for comments just to be safe.